index.html

```diff
@@ -18,8 +18,6 @@
     <!-- javascript -->
     <script defer src="./js/menuDisplay.js"></script>
     <script defer src="./js/poppup.js"></script>
```

Comment:
Great job @AbigiyaTY 🤝

- [ ] You may reduce the `font weight` of your text on the mobile popup so that your design looks similar to the one in the Figma template.
- [ ] You may as well add a `margin-bottom` to your mobile popup so that it looks like the Figma design provided.
- [ ] You may also align your languages and buttons accordingly. As your design appears a bit different from the design provided.
- Your design
![mobile design popup](https://user-images.githubusercontent.com/63917725/180784746-a9583ffb-beb9-4f21-b080-20b3172b0e79.PNG)

- Figma design
![Figma mobile design](https://user-images.githubusercontent.com/63917725/180784838-e3e2fe9a-138d-4cfb-9d4d-185f240e6047.PNG)